terraform/azure/main.tf

```diff
@@ -1,35 +1,49 @@
 terraform {
   backend "azurerm" {
-    resource_group_name  = "orso-global-rg"
-    storage_account_name = "orsoglobalsa"
+    resource_group_name  = "${var.club}-global-rg"
```

Comment:
why does terraform complain:

>  Error: Variables not allowed
> │
> │   on main.tf line 3, in terraform:
> │    3:     resource_group_name  = "${var.club}-global-rg"
> │
> │ Variables may not be used here.

Reply:
you cannot use variables in azurerm, because this is used in "terraform init" even before the variables are known and used in the plan step.